content/en/docs/Installation/_index.md

```diff
@@ -439,7 +447,22 @@ helm search repo kyverno
 Run the upgrade command picking the target version.
 
 ```sh
-helm upgrade kyverno --namespace kyverno kyverno/kyverno --version <version_number>
+helm upgrade kyverno-crds kyverno/kyverno-crds --namespace kyverno --version <version_number>
+helm upgrade kyverno kyverno/kyverno --namespace kyverno --version <version_number>
+```
+
+{{% alert title="Note" color="warning" %}}
```

Comment:
This is going to be a real big pain in the ass for users. Is there a one-liner we can give them to "back up" these policies prior to upgrading?

Reply:
I added some basic backup/restore commands